lib/order.rb

```diff
@@ -0,0 +1,20 @@
+class Order
+  attr_reader :grand_total
+
+  def initialize
+    @lines = "Thanks for your order\n"
+    @grand_total = 0
+  end
+
+  def add_line line_item
+    sub_total = line_item.sub_total
+    dish_name = line_item.dish
+    dish_qty = line_item.quantity
+    @grand_total += line_item.sub_total
+    @lines << "#{dish_qty} x #{dish_name} = £#{sub_total}\n"
+  end
+
```

Comment:
Same thing like in Menu class ... we should be dealing with objects and not just some strings.